lib/const.rb

```diff
@@ -50,31 +50,31 @@
 # Glibc version can be found from the output of libc.so.6
 LIBC_VERSION = `/#{ARCH_LIB}/libc.so.6`[/Gentoo ([^-]+)/, 1]
 
-if ENV['CREW_PREFIX'].to_s.empty? || (ENV['CREW_PREFIX'] == '/usr/local')
-  CREW_BUILD_FROM_SOURCE = ENV.fetch('CREW_BUILD_FROM_SOURCE', nil)
-  CREW_PREFIX = '/usr/local'
+CREW_PREFIX = ENV.fetch('CREW_PREFIX', '/usr/local')
+
+if CREW_PREFIX == '/usr/local'
+  CREW_BUILD_FROM_SOURCE = !!ENV['CREW_BUILD_FROM_SOURCE']&.eql?('1')
   HOME = Dir.home
 else
-  CREW_BUILD_FROM_SOURCE = 1
-  CREW_PREFIX = ENV.fetch('CREW_PREFIX', nil)
+  CREW_BUILD_FROM_SOURCE = true
   HOME = CREW_PREFIX + Dir.home
 end
 
 CREW_ARCHIVE_DEST = ENV.fetch('CREW_ARCHIVE_DEST', Dir.pwd)
 
-CREW_IN_CONTAINER = File.exist?('/.dockerenv') || !ENV['CREW_IN_CONTAINER'].to_s.empty?
+CREW_IN_CONTAINER = File.exist?('/.dockerenv') || !!ENV['CREW_IN_CONTAINER']&.eql?('1')
```

Comment:
`|| ENV['CREW_IN_CONTAINER']`?

Reply:
All boolean constants will switch to true only if `ENV['const_name'] == '1'` for simplicity now.